src/subdataframe/subdataframe.jl

```diff
@@ -136,13 +136,16 @@ end
 
 Base.copy(sdf::SubDataFrame) = parent(sdf)[rows(sdf), parentcols(index(sdf), :)]
 
-function without(df::SubDataFrame, icols::Vector{<:Integer})
```

Comment:
Why remove this?

Reply:
it is redundant with `without` in abstractdataframe/abstractdataframe.jl